lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/interactor/usecases/get_n_profiles_use_case.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:seeds/datasource/remote/api/profile_repository.dart';
+import 'package:seeds/datasource/remote/model/profile_model.dart';
+import 'package:seeds/domain-shared/base_use_case.dart';
+
+class GetNProfilesUseCase extends InputListUseCase<ProfileModel, _Input> {
```

Comment:
Get Multiple profiles: GetNProfilesUseCase -> GetProfilesUseCase
Get Single profile: GetProfileUseCase

I think we have a shared use case that gets profiles by "ids" already, this is a very common thing we do in multiple places in the app? 

@gguij004  do we have this Use case? 

If not. we should have a shared use case that does this.

Reply:
I can give it a more significant name.
But this cannot be a use case for a given list of accounts ids.
I only need up to 3 profiles if the list contains more only take 3, if it contains less ask 1 or 2, that is why I use the `take` method

Reply:
this is good now, I meant we dont need another base use case. Which you removed. 👍